charts/sourcegraph/examples/external-redis/override-separate-configmap.yaml

```diff
@@ -0,0 +1,86 @@
+# Demonstrate using external redis instance(s)
```

Comment:
What is the benefit of showing the exact same setup, with both a secret and a configmap? There's no significant difference between them, and it means more maintenance in the future.

Reply:
> What is the benefit of showing the exact same setup, with both a secret and a configmap? There's no significant difference between them, and it means more maintenance in the future.

🧐 I guess adding a reference k8s doc should be sufficient enough

still figuring out the right balance of how much k8s knowledge we should cover in docs/examples.